pkgs/build-support/docker/default.nix

```diff
@@ -1105,7 +1111,7 @@ rec {
             };
             nativeBuildInputs = [ makeWrapper ];
           } ''
-          makeWrapper ${streamScript} $out --add-flags ${conf}
+          makeWrapper $streamScript $out --add-flags $conf
```

Comment:
What does this change do?

Reply:
It uses the environment variable passed to the derivation environment using the attrset above, instead of interpolating it into this string that runs as bash commands.

Reply:
Indeed, but to what end? Why was this change made?

If I had to pick between the two, isn't interpolation slightly better because it avoids polluting the environment and any possible clashes with `mkDerivation` internals?

Reply:
Having `streamScript` and `conf` in the environment instead of interpolated means we can use `.overrideAttrs (o: { streamScript = ... })` to override the stream script, and `.conf` to extract the conf to use for something else.
In practice, this means I won't have to parse it out of the buildCommand anymore.

Reply:
We may switch `conf` to the `nix2container` format in the future.